### PR TITLE
Add a new friendly civilian hail

### PR DESCRIPTION
**Pull request by petervdmeer**
Added a new friendly civilian hail. This hail was hanging around in one of my plugins, but is better placed in the core game if it is good enough to be included there.

### CODE REVIEW

**Comment by jafdy** [comment]
`Hey, I mailed me this ship.` Is a very possible statement with this layout. (I'm sure there are some other just as odd ones in there but I'm in a rush today.)

**Comment by ZBok** [comment]
"Hey look at me; my talking cat built my planet this gold plated alien style money maker!"

Gold.

**Comment by petervdmeer** [comment]
@jafdy I think I removed most strange constructs.
@ZBok Sorry, non-native speaker. Missed the alternative meaning. Fixed it (I hope).

**Comment by ZBok** [comment]
@petervdmeer were you referring to "money maker? I think that phrase is ok, I actually just thought that particular sentence was, although quite funny, a little odd :D

It seems to state that the talking cat, built a spaceship (interesting in itself), for a planet.

File: data/hails.txt [hunk 1]
```diff
@@ -1025,6 +1025,92 @@ phrase "friendly civilian"
...
+		"my planet "
+		"my sister "
+		"my parents "
+		"my wife "
+		"my ex-wife "
+	word
+		"this "
+	word
+		""
+		"beautifull "
```
[the remark this answers is not in the document]
Comment (petervdmeer) [reply data/hails.txt]: Fixed typo and moved spaces to start of line.

Removed "Look at me" entry -> It is indeed odd to say for somebody showing off his spaceship (and we already have "look at my ship").

Removed "mailed". -> I was thinking of a delivery service similiar to Amazon package delivery (but then the size of spaceships, large spaceships delivering small ones). But now I realize that mailing is not the correct word for that. (Added delivered).

Removed negative words for spaceships -> A well-armed, expensive piece of junk is a strange combination indeed.

Removed "mother in law". -> Mother-in-law jokes are not really appropriate for Endless Sky. 
Kept the "my planet" entry. -> There are planetary defense fleets in the game. So building, trading or ordering spaceships for a planet is not too uncommon.

Kept "talking cat getting spaceship". -> The game has creatures that look like squirrels and birds building spaceships. People might also own uncommon catlike creatures.

Kept the homeless guy, but added that he won the lottery. (It is a common story in many countries that some homeless guys are actually quite rich.)

Please let me know if you still see strange or silly combinations that don't work.

**Comment by Pointedstick** [comment]
"See this vessel? You know that my talking cat built my planet this strange cheap passenger ferry."

I dunno, I'm still not feeling it

**Comment by petervdmeer** [comment]
> "See this vessel? You know that my talking cat built my planet this strange cheap passenger ferry."
> 
> I dunno, I'm still not feeling it
You're right. That one doesn't work since the second part should then end in a question mark. I removed this beginning. And the planet part also doesn't seem to work as I intended, also removed that one.

But maybe this new hail just isn't going to fly.
Shall I cancel/close the pull request?

**Comment by petervdmeer** [comment]
Cancelling pull request; this hail is probably not going to fly and is not critical at all for the game. Let's focus on other pull requests that add more value.

**Comment by Pointedstick** [comment]
Best of luck with your next PR! :)

**Pull request closed without merging**